### DISCUSSION TITLE
How to add sentry to trpc?

### DISCUSSION THREAD

**Discussion by Tobjoern** [trpc/trpc#4596] [General]
Context:
I built a next.js app in a monorepo, with the backend trpc project being in a separate package.
I import the trpc project into /trpc/[trpc].ts file.

Now I'm looking to add sentry to the project, I already added it to the basic next.js project (https://docs.sentry.io/platforms/javascript/guides/nextjs/), but this doesn't capture the context for errors thrown on the server side.
My approach going forward would be to install 'https://docs.sentry.io/platforms/node' inside the trpc project.
                                                                                                                               
I'm probably not the first person to have this issue, is there a guide for this?

**Comment by MrRoiz** [comment]
Hey, were you able to solve it? getting the same issue here.

**Comment by keyserj** [comment]
Have you tried the sentry trpc middleware? https://docs.sentry.io/platforms/javascript/guides/nextjs/configuration/integrations/trpc/
